ElectronicObserver/Window/Control/Paging/PagingControlView.xaml

```diff
@@ -0,0 +1,32 @@
+﻿<UserControl
+	x:Class="ElectronicObserver.Window.Control.Paging.PagingControlView"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:local="clr-namespace:ElectronicObserver.Window.Control.Paging"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:sys="clr-namespace:System;assembly=mscorlib"
+	xmlns:ui="http://schemas.modernwpf.com/2019"
+	x:Name="PagingControlViewElement"
+	d:DesignHeight="450"
+	d:DesignWidth="800"
+	mc:Ignorable="d"
+	>
+	<StackPanel DataContext="{Binding ViewModel, ElementName=PagingControlViewElement}" Orientation="Horizontal">
+
+		<TextBlock Margin="5" VerticalAlignment="Center">
+			<Run Text="{Binding PagingControl.Page, Mode=OneWay}" />
```

Comment:
Is there any need to write `Page` here? I think it should be obvious enough without it. Especially since there's `Items per page` on the right side anyway.

Reply:
Removed from xaml, resources and translation viewmodel